### DISCUSSION TITLE
model overfitting

### DISCUSSION THREAD

**Discussion by pedrozamboni** [neuralhydrology/neuralhydrology#118] [Q&A]
I'm training a model with a dataset that has 735 watersheds, of which 514 I'm using for training, 110 for validation and 111 for testing. I'm having problems with overfitting the validation set. I'm using linear activation with AdamW and weight_decay of 10e-5. Any tips on how to reduce the overfitting problem?

**Comment by kratzert** [comment]
Not really and without you sharing more information, it would be hard to help, sorry.

**Reply by pedrozamboni** [reply]
Have you already implemented a basin normalization in the code?

**Reply by kratzert** [reply]
WDYM with basin normalization? THat you want to normalize the labels for each basin separately instead of normalizing the labels with the global mean/std?
